src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -151,13 +161,25 @@ for buttons! Button elements provide accessible behaviour and built-in form
 submission functionality, and they can easily be styled. There's no point in
 using a `<div>` or some other element pretending to be a button. 
 
-Give the submit button in a sign-up or sign-in form text content that says what it
-does, such as **Create account** or **Sign in**, not **Submit** or **Start**.
+Ensure that the submit button says what it does. Examples include **Create account** or 
+**Sign in**, not **Submit** or **Start**.
+
+### Ensure successful form submission {: #submission }
+
+Help password managers understand that a form has been submitted. There are two 
+ways to do that: 
+
+* Navigate to a different page.
+* Emulate navigation with `History.pushState()` or `History.replaceState()`, 
+and remove the password form.
+
+If you use `XMLHttpRequest` or `fetch`, make sure that sign-in success or failure 
+is reported in the response.
```

Comment:
I don't know what this means and how it would be used.

We have a heuristic in which we consider a form submission to happen if we first (!) see an XMLHttpRequest / fetch and then the form is taken out of the DOM.

BTW. Chrome follows two steps 1) did we see a form submission (all the items above apply) and 2) was the form submission successful (we consider it successful if the landing page does not show the same form again).

Reply:
I've changed the text to the following: 'With an `XMLHttpRequest` or `fetch` request, make sure that sign-in success is reported in the response and handled by taking the form out of the DOM as well as indicating success to the user.'